kitchen.yml

```diff
@@ -15,6 +16,11 @@ platforms:
   - name: centos-7
   - name: debian-8
   - name: ubuntu-16.04
+  - name: fedora-30
+    driver:
+      provision: true
+      vagrantfiles:
+        - vagrant/fedora30.rb
```

Comment:
👎  I'm super not OK with us managing our own Vagrantfiles here. If it isn't available upstream in bento please add it threre.

Reply:
Done. There is no Fedora 30 bento yet so it should've been 29 in the first place thinking about it, my bad.

Reply:
Ace thanks.